ftw/subsite/browser/configure.zcml

```diff
@@ -6,18 +6,6 @@
 
   <include package="plone.app.contentmenu" />
 
-
-	<!-- dashboard for the front-pages -->
-
- <browser:resourceDirectory
```

Comment:
Did you remove this by accident, or is it no longer in use? if so what's inside?

Reply:
I did it by mistake. It looked like one block because of the indentation and I was to remove the second part.